packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -366,6 +348,16 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
      *          INTERNAL FUNCTIONS          *
      ****************************************/
 
+    function _computePendingStakes(address wallet, uint256 amount) internal virtual {}
+
+    function _setPendingStake(
```

Comment:
seeing this is: a) only called here within `Staker.sol` b) only updates a single tracker does it not make sense to just do this inline on line 770?

Reply:
o nvm you use this within the Votingv2 contract. disregard this comment.